frontend/src/forms/CustomerForm.jsx

```diff
@@ -59,6 +59,10 @@ export default function CustomerForm({ isUpdateForm = false }) {
             required: true,
             message: 'Please input your phone!',
           },
+          {
+            pattern: /^(?:[+\d()\-\s]+)$/,
```

Comment:
Seeing as we reuse this regexp twice, it might be helpful to put it into a helper file and add a comment as to where the regexp came from.